src/components/atoms/_OcGhostElement/_OcGhostElement.vue

```diff
@@ -28,10 +41,41 @@ export default {
       return this.previewItems.length
     },
   },
+  methods: {
+    getStyleClass(index) {
```

Comment:
naming is not ideal, since you return `styles`, not `classes`.

Reply:
yes